.changeset/thirty-cats-help.md

```diff
@@ -0,0 +1,46 @@
+---
+'@backstage/plugin-catalog-backend': patch
+---
+
+Change script in **UrlReaderProcessor.ts** adding some code to handle URL Reader from GCS with wildcard \*
+
+```diff
```

Comment:
You can remove this `diff` and instead just summarize the change and what it does. This text makes it to the changelog for each package, so take some inspiration from these messages: https://github.com/backstage/backstage/blob/master/plugins/scaffolder-backend/CHANGELOG.md#patch-changes-3

Reply:
@benjdlambert  Already summarized. Thank you